sbpy/data/orbit.py

```diff
@@ -865,10 +864,12 @@ def D_criterion(self, obj, version='sh'):
 
             if version.lower() == 'sh':
                 # Southworth-Hawkins function
+                register(self.D_criterion, {'method': '1963SCoA....7..261S'})
                 d2 = diff_e**2 + diff_q**2 + 4 * sin_i2 \
                     + (sum_e * np.sin(pi_ba / 2))**2
             else:
```

Comment:
Can this method raise an error if `version` is not valid?

Reply:
Yes it does.  Lines 827 - 829 checks the value of `version` and return an exception if not one of the three.

This reminds me:  Do we have a convention in `sbpy` to raise `SbpyException` for errors?  Right now it raises a generic `ValueError`.